src/CadViewerJscad.tsx

```diff
@@ -119,6 +113,17 @@ export const CadViewerJscad = forwardRef<
       }
     }, [internalCircuitJson])
 
+    const pcbThickness = useMemo(() => {
+      if (!internalCircuitJson) return 1.2
+      try {
+        const board = su(internalCircuitJson as any).pcb_board.list()[0]
+        // Default to 1.2 to match BoardGeomBuilder default
+        return board?.thickness ?? 1.2
+      } catch (e) {
+        return 1.2
+      }
+    }, [internalCircuitJson])
```

Comment:
try to avoid top level pollution, move this to a method (the LoC on this file is very precious)